dbms/src/Storages/Page/V3/WAL/WALReader.cpp

```diff
@@ -209,7 +209,7 @@ bool WALStoreReader::openNextFile()
         auto read_buf = createReadBufferFromFileBaseByFileProvider(
             provider,
             fullname,
-            EncryptionPath{parent_path, filename},
+            EncryptionPath{parent_path + "/" + filename, ""},
```

Comment:
Make the L206 to be `const auto fullname = next_file.fullname(next_file.stage)` and make the L212 to be `EncryptionPath{fullname, ""},` may be helpful to elimate the differnece?
https://github.com/pingcap/tiflash/blob/f4ca432d741be05df8066f9cee9fe195c8956c5d/dbms/src/Storages/Page/V3/WALStore.cpp#L132-L140